config/sync/views.view.eif_related_entities.yml

```diff
@@ -0,0 +1,233 @@
+uuid: e839e08e-342b-4f8c-882d-94ca8da43ec8
+langcode: en
+status: true
+dependencies:
+  config:
+    - search_api.index.published
+  module:
+    - joinup_core
```

Comment:
I verified this, and this indeed depends on `joinup_core` because it is using the `Tiles` views style.